torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -1651,7 +1651,14 @@ def codegen_multi_output(self, name, value):
         pass
 
     def codegen_subgraph_prefix(self, subgraph, outer_inputs, outer_outputs):
-        for inner_input, outer_input in zip(subgraph.graph.graph_inputs, outer_inputs):
+        assert len(subgraph.graph.graph_inputs) == len(outer_inputs)
+
+        for (inner_input, inner_input_val), outer_input in zip(
+            subgraph.graph.graph_inputs.items(), outer_inputs
+        ):
+            if not isinstance(inner_input_val, ir.TensorBox):
```

Comment:
Wondering if it must be `ir.TensorBox` in all cases and not just any `ir.Buffer`?